### PR TITLE
add Phone#country_code method

### PR DESCRIPTION
**Pull request by mastahyeti**
This PR adds a `country_code` method to the `Phone` class.

### CODE REVIEW

**Comment by daddyz** [comment]
@mastahyeti Is there any reason why you need unformatted national number? There is a method `national` which returns formatted national number.

File: lib/phonelib/phone.rb [hunk 1]
```diff
@@ -78,6 +79,11 @@ def country
       end
     end
 
+    # Returns the country code from the original phone number.
+    def country_code
+      @data.values.first[:country_code]
```
Comment (daddyz) [inline lib/phonelib/phone.rb]: @mastahyeti Can you base this result on `country` method and please include a check if there is no valid country with test for it, thanks

**Comment by mastahyeti** [comment]
> @mastahyeti Is there any reason why you need unformatted national number? There is a method national which returns formatted national number.

Nope. `national` will work fine. I'll revert that second commit.

**Comment by daddyz** [comment]
@mastahyeti thanks

**Pull request merged**